lightkurve/collections.py

```diff
@@ -95,6 +188,36 @@ class LightCurveFileCollection(Collection):
     def __init__(self, lightcurvefiles):
         super(LightCurveFileCollection, self).__init__(lightcurvefiles)
 
+    @property
+    def PDCSAP_FLUX(self):
+        return LightCurveCollection([lcf.PDCSAP_FLUX for lcf in self])
+
+    @property
+    def SAP_FLUX(self):
+        return LightCurveCollection([lcf.SAP_FLUX for lcf in self])
+
+    def stitch(self, offset=0.1, **kwargs):
+        try:
+            warnings.warn('Stitching a `LightCurveFileCollection` which contains both SAP and '
+                         'PDCSAP_FLUX. Plotting PDCSAP_FLUX. You can remove this warning by '
+                         'using `LightCurveFileCollection.PDCSAP_FLUX.stitch()`.',
+                         LightkurveWarning)
```

Comment:
I'm worried that warnings like this will make people insensitive to warnings.  Do we really need to alert the user here?

Reply:
Yes because we're making an assumption that they want PDCSAP_FLUX when we'd rather they were explicit.